akka-http/src/main/resources/reference.conf

```diff
@@ -39,5 +39,5 @@ akka.http.routing {
 
   # Fully qualified config path which holds the dispatcher configuration
   # to be used by FlowMaterialiser when creating Actors for IO operations.
-  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher}
+  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher} // TODO point to akka.stream.file-io-dispatcher instead?
```

Comment:
okey, linked to `akka.stream.default-file-io-dispatcher` then, it's default conf is:

```
    default-file-io-dispatcher {
      type = "Dispatcher"
      executor = "thread-pool-executor"
      throughput = 1

      thread-pool-executor {
        core-pool-size-min = 2
        core-pool-size-factor = 2.0
        core-pool-size-max = 16
      }
    }
```

The `throughput = 1` we thought makes sense for a "file" worker threadpool. Let me know if this can be improved as a nice default.